pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/jstl/CustomTypeConverter.java

```diff
@@ -394,4 +413,25 @@ protected Instant coerceToInstant(Object value) {
     }
     throw new IllegalArgumentException("Cannot convert type " + value.getClass() + " to Instant");
   }
+
+  public <T> T convert(Object value, Class<T> type) throws ELException {
```

Comment:
There are few options here:
1) Use two different impls, one for compute and one for cast and the common conversion can go in an abstract method
2) Stick to the convertToType interface method and have it deal with null context.

Reply:
I used a variant of option 1 in the latest path where the Jstl conversions return null and direct converter invocation (e.g. from cast step or utility methods) overrides the Jstl class and handles the null by returning `ELSupport.coerce*` methods